test/e2e/chaos/podchaos/pod_failure.go

```diff
@@ -164,7 +164,7 @@ func TestcasePodFailurePauseThenUnPause(ns string, kubeCli kubernetes.Interface,
 
 	By("waiting for assertion some pod fall into failure")
 	// check whether the pod failure chaos succeeded or not
-	err = wait.PollImmediate(5*time.Second, 5*time.Minute, func() (done bool, err error) {
+	err = wait.Poll(5*time.Second, 5*time.Minute, func() (done bool, err error) {
```

Comment:
1. when creating chaos, will apply chaos first and then update chaos to K8s. create -> apply chaos -> update1
2. will pause chaos by update annotation of chaos to K8s at L184.  pause -> update2 -> pause chaos
 
the execution order may be: create -> apply chaos -> pause -> update2 -> update1
and update1 will get error 
```log
cannot be updated: the object has been modified; please apply your changes to the latest version and try again
```

so change `PollImmediate` to `Poll` to sleep 5 second

Reply:
similar reason for L101